### ISSUE TITLE
Import names into `enterprise`?

### ISSUE THREAD

**Issue by vallis** [nanograv/enterprise#110]
I like the logical arrangement of submodules within enterprise, but it leads to some verbosity in analysis scripts. For instance, having to `import enterprise.white_signals as white_signals`, and then write `white_signals.MeasurementNoise`, etc.

We could alleviate this to different degrees: e.g.:
- Import all names into `enterprise`. I think the risk of collisions is actually slight, but one may prefer the structuredness of separating parameters from signals, etc.
- Import all signal names into `enterprise.signal`, so users don't need to think about the type of a signal.
- Import all utility functions into `enterprise` proper.

**Comment by jellis18** [comment]
Yes this is definitely work discussing. I had assumed we would clean this up once we got the basics in place and I suppose it is now time for this.
